src/GitHub.UI/Helpers/SharedDictionaryManagerBase.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Windows;
+using System.Collections.Generic;
+
+namespace GitHub.Helpers
+{
+    public class SharedDictionaryManagerBase : ResourceDictionary
```

Comment:
Given that this class needs to manage disposable resources, it should be `IDisposable` (if you have a `CompositeDisposable` instead of a `List<IDisposable>`, the code analysis will let you know this 😉